src/component/Clock.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ date: new Date() });
+      // eslint-disable-next-line no-console
+      console.info(this.state.date.toUTCString().slice(-12, -4));
```

Comment:
will be better to create separate function for transforming date, for now you used the same code twice